src/components/Dialogs/Message/AddMessageForm.js

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
 import { Field, reduxForm } from 'redux-form';
+import { useTranslation } from 'react-i18next';
 
-import { I18N } from '../../../utils/constants';
+import { required, maxLengthCreator } from '../../../utils/validators';
+import { Textarea } from '../../../utils/FormControl/formsControls';
+
+const maxLenght10 = maxLengthCreator(10);
 
 const AddMessageForm = (props) => {
+  const { t } = useTranslation();
   const { handleSubmit } = props;
+
   return (
     <form onSubmit={handleSubmit}>
       <div>
-        <Field component="textarea" name="newMessageBody" />
+        <Field component={Textarea} name="newMessageBody" validate={[required, maxLenght10]} />
       </div>
       <div>
-        <button>{I18N.EN.SEND}</button>
+        <button>{t('SEND')}</button>
```

Comment:
Even with added third-party translation library it is still useful to store the translation keys in some kind of I18N object.

Reply:
to the current moment, I would prefer to use the logic implying by the library that the locale itself contains nothing more than word translation

Reply:
What if you send a wrong key? Will it display anything? Will it break?
By bringing in the localisation library and store all the translation in one place.
However, the current line expects that the string you passed is registered in both JSON - so it's better to store all these "registered and translated" pieces in one place close to the actual translations JSONs.